### PR TITLE
Improve the code highlight style

### LINKED ISSUE #8407: New docs: Code view needs adjustment
### Description

Code view needs adjustment of point 6. on page https://handsontable.com/docs/internationalization-i18n/#language-file needs adjustment, especially on dark mode. 

Screenshot 2021-07-09 at 09 13 41

On the older version, this code looks like diff.
'+' was visible https://handsontable.com/docs/8.4.0/tutorial-internationalization.html

### PR DESCRIPTION
**Pull request by budnix**
### Context

The PR improves the code highlight by adding the plus sign in the highlighted lines (as in the https://handsontable.com/docs/8.4.0/tutorial-internationalization.html#available-languages) and changes the background color.

Before (old docs engine):
image

Before (new docs engine):
image

After:
image

_[skip changelog]_

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. fixes #8407
2.
3.

### Affected project(s):
- [ ] `handsontable`
- [ ] `@handsontable/angular`
- [ ] `@handsontable/react`
- [ ] `@handsontable/vue`

### Checklist:

- [x] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project.
- [x] I have signed the [Contributor License Agreement](https://docs.google.com/forms/d/e/1FAIpQLScpMq4swMelvw3-onxC8Jl29m0fVp5hpf7d1yQVklqVjGjWGA/viewform?c=0&w=1)
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Review by wszymanski — APPROVED** [review]
It looks good!

**Review by wojciechczerniak — CHANGES_REQUESTED** [review]
I agree that it looks good, but IMO we will lose code highlighting ability if we do it this way.

Code highlighting may be very useful when writing the docs to focus the reader on the essential code fragments ie. ignoring imports, class boilerplate, showing (highlighting as name says) the method body. Without adding comments to get the reader attention `// this matters`

On the other hand, tutorials would benefit from diff's like that, making it clear that lines were added. So we may take a little longer to find the solution and keep both.

Prism.js has special ```diff tag: https://prismjs.com/test.html#language=diff&text=import%20Handsontable%20from%20'handsontable'%0A%2B%20import%20esVE%20from%20'.%2Fes-ve'%0Aimport%20enGB%20from%20'.%2Fen-gb' maybe it will be better suited for this use case? It has one caveat tho, doesn't highlight the JS code as normal JS tag does

WDYT?

**Comment by budnix** [comment]
> I agree that it looks good, but IMO we will lose code highlighting ability if we do it this way.
> 
> Code highlighting may be very useful when writing the docs to focus the reader on the essential code fragments ie. ignoring imports, class boilerplate, showing (highlighting as name says) the method body. Without adding comments to get the reader attention `// this matters`

> On the other hand, tutorials would benefit from diff's like that, making it clear that lines were added. So we may take a little longer to find the solution and keep both.
> 
> Prism.js has special ```diff tag: https://prismjs.com/test.html#language=diff&text=import%20Handsontable%20from%20'handsontable'%0A%2B%20import%20esVE%20from%20'.%2Fes-ve'%0Aimport%20enGB%20from%20'.%2Fen-gb' maybe it will be better suited for this use case? It has one caveat tho, doesn't highlight the JS code as normal JS tag does
> 
> WDYT?

I think that the highlighting, in that case, does the job. This is the only place where we use it so I would leave it as it is. If we use more diff-like examples in the future, we may think about improving that. Is that sound ok to you?

**Comment by wojciechczerniak** [comment]
No, not really. If we change the highlighting no one will ever use it for other purpose than diff'ing. We're taking a tool from technical writing toolset and changing it into something else.

Highlight is enough in this case and is a more universal.

File: docs/.vuepress/theme/styles/code-group.styl [hunk 1]
```diff
@@ -24,3 +24,14 @@
 div[class*="language-"]::before {
   content: '';
 }
+
+div[class*="language-"] .highlight-lines {
```
Comment (wojciechczerniak) [inline docs/.vuepress/theme/styles/code-group.styl:28]: Is there a way to add custom class to example block? 

```suggestion
div[class*="language-"] .highlight-lines.highlight-diff {
```

Usage
```
```example { highlight={2,4} class=.highlight-diff }

```

**Comment by budnix** [comment]
> No, not really. If we change the highlighting no one will ever use it for other purpose than diff'ing. We're taking a tool from technical writing toolset and changing it into something else.
> 
> Highlight is enough in this case and is a more universal.

It's not easy to implement a new syntax for diffing like `js { highlight={2,4} class=.highlight-diff }`. I haven't found any interesting plugin for markdown-it. 

At this point, I don't know what to do with the task. The PR is about improving the highlighting syntax, so I restored the original look based on the older engine. I suggest closing the PR or accept the with the changes in styles and then create a separate issue with "diff" ideas WDYT?

**Comment by budnix** [comment]
After discussion with @wojciechczerniak, we both agreed that we want to leave the original Vuepress highlight color.

**Pull request closed without merging**